docs/schema/records_reference.md

```diff
@@ -28,81 +13,112 @@ The versioned release format is designed to support analysis of how data has cha
 ```
 
-## Package Metadata
+## Package metadata
 
-Records must be published within a [record package](record_package).
+Records must be published within a [record package](record_package). The record package provides metadata about the record(s) that it contains.
 
 ```eval_rst
 .. jsonschema:: ../../build/current_lang/record-package-schema.json
     :include:
     :collapse: records
-
 ```
 
-See the guidance on [package metadata](../reference/#package-metadata). In addition, a record package includes:
+See the [licensing guidance](../../guidance/publish/#license-your-data) for more details on selecting a license, and publishing license information.
 
-* `packages` - links to the release packages used to compile the records
-* `records` - one or more records, each of which, at least, links to or embeds the releases about the contracting process that it describes
+See the [publication policy](../../guidance/publish/#finalize-your-publication-policy) guidance for more details on what to include in a publication policy.
 
-You can see this in the full example of a record below.
+The record package metadata has two differences from the release package metadata:
+
+* Instead of a `releases` array, a record package has a `records` array containing one or more records.
+* A record package has a `packages` array, to link to any release packages that were used to prepare the records.
+
+The following example demonstrates all package metadata and record fields.
 
 ```eval_rst
 .. jsoninclude:: ../examples/merging/versioned.json
    :jsonpointer:
    :expand: packages, records
-   :title: Full_record
-
+   :title: package
 ```
 
 ## Record structure
 
-Each record package contains an array of one or more records, consisting of the following sections:
+A record **must** contain an [ocid](../identifiers/#ocid) and all [releases](#releases) about the contracting process. As such, a record functions as an index of all releases about a contracting process.
+
+A record **should** contain a [compiledRelease](#compiled-release) object, which represents the state of the contracting process at the time of the record's publication.
 
-* Releases (required)
-* Compiled Release (recommended)
-* Versioned Release (optional)
+A record **may** contain a [versionedRelease](#versioned-release) object, which aggregates all values from all releases in a single object. The versioned release is designed to make it easy to see how values change from one release to another, and will often be generated by data users, rather than by publishers.
 
-### Release list
+### Releases
 
-The releases that make up a contracting process can be provided in two ways:
+Each release in a record can be provided as either a linked release or an embedded release.
 
-* URLs and metadata for each release; or
-* Embedded copies of the release;
+#### Linked releases
+
+A linked release follows a simple schema:
+
+```eval_rst
```

Comment:
@jpmckinney the first row of the table is missing the title/description 
![image](https://user-images.githubusercontent.com/6834122/84403250-7bb30380-abd3-11ea-839b-e5a3a0588c66.png)

Reply:
Thanks. I'll fix the JSON Pointer so that the table starts at the `url` row and omits the `/0`.